build.gradle

```diff
@@ -135,7 +135,7 @@ switch (DefaultPlatform.RESOURCE_PREFIX) {
         ext.nativeRustTarget = 'linux-x86-64'
         break
     case 'win32-x86-64':
-        ext.nativeRustTarget = 'win32-x86-64-gnu'
+        ext.nativeRustTarget = 'win32-x86-64-msvc'
```

Comment:
this breaks CI

Reply:
Let's revert that and maybe open a bug so we can handle that afterwards?

Reply:
Eugh, sorry, this was not intentional. Let me revert changes to this file.